README.md

```diff
@@ -42,27 +42,46 @@ to integrate ATT&CK more easily and consistently into their products.
 
 ## Installation
 
-1. Get Docker: <https://docs.docker.com/get-docker/>
-2. Get Docker Compose:  <https://docs.docker.com/compose/install/>
-3. Ensure Docker is running. On some operating systems (e.g., MacOS), you will
-   need to provide Docker with permissions before proceeding.
-4. Download docker-compose.yml (view raw, save as)
-
-    ```url
-    https://github.com/center-for-threat-informed-defense/tram/blob/master/docker/docker-compose.yml
-    ```
-
-5. If desired, edit the settings in `docker-compose.yml`
-6. Navigate to the directory where you saved `docker-compose.yml`
-7. Run TRAM using docker
+1. Install Docker tools:
+    * Docker: <https://docs.docker.com/get-docker/>
+    * Docker Compose: <https://docs.docker.com/compose/install/>
+    * Verify that Docker is running by running `docker ps` from a shell. If it
+      shows, "CONTAINER ID   IMAGE     COMMAND" on the first line, then it is
+      running. If it says, "cannot connect to Docker daemon," then Docker is not
+      running.
+2. Download docker-compose.yml for TRAM, using [this
+   link](https://raw.githubusercontent.com/center-for-threat-informed-defense/tram/master/docker/docker-compose.yml)
+   or using curl:
 
     ```shell
-    docker-compose -f docker-compose.yml up
+    $ curl -O https://raw.githubusercontent.com/center-for-threat-informed-defense/tram/master/docker/docker-compose.yml
```

Comment:
Really great overall, while you are here, can you update the links to use permalinks? Either to the a specific release (https://github.com/center-for-threat-informed-defense/tram/releases/download/v1.1.2/docker-compose.yml) or a github permalink (https://github.com/center-for-threat-informed-defense/tram/blob/b7268eb5333696ae753d58e7dbb1860293823767/docker/docker-compose.yml for example), so if we reorg the repo or something the links won't break?